connection.go

```diff
@@ -31,58 +39,6 @@ type ExchangeDeclareSettings struct {
 	Args amqp.Table
 }
 
-// QueueDeclareSettings is the settings that will be used when the response
-// any kind of queue is declared. Se documentation for amqp.QueueDeclare
-// for more information about these settings.
-type QueueDeclareSettings struct {
-	// DeleteWhenUnused sets the auto-delete flag. It's recommended to have this
-	// set to false so that amqp-rpc can reconnect and use the same queue while
-	// keeping any messages in the queue.
-	DeleteWhenUnused bool
-
-	// Durable sets the durable flag. It's recommended to have this set to false
-	// and instead use ha-mode for queues and messages.
-	Durable bool
-
-	// Exclusive sets the exclusive flag when declaring queues. This flag has
-	// no effect on Clients reply-to queues which are never exclusive so it
-	// can support reconnects properly.
-	Exclusive bool
-
-	// Args sets the arguments table used.
-	Args amqp.Table
-}
-
-// ConsumeSettings is the settings that will be used when the consumption
-// on a specified queue is started.
-type ConsumeSettings struct {
-	// Consumer sets the consumer tag used when consuming.
-	Consumer string
-
-	// AutoAck sets the auto-ack flag. When this is set to false, you must
-	// manually ack any deliveries. This is always true for the Client when
-	// consuming replies.
-	AutoAck bool
-
-	// Exclusive sets the exclusive flag. When this is set to true, no other
-	// instances can consume from a given queue. This has no affect on the
-	// Client when consuming replies where it's always set to true so that no
-	// two clients can consume from the same reply-to queue.
-	Exclusive bool
-
-	// QoSPrefetchCount sets the prefetch-count. Set this to a value to ensure
-	// that amqp-rpc won't prefetch all messages in the queue. This has no
-	// effect on the Client which will always try to fetch everything.
-	QoSPrefetchCount int
-
-	// QoSPrefetchSize sets the prefetch-size. Set this to a value to ensure
-	// that amqp-rpc won't prefetch all messages in the queue.
-	QoSPrefetchSize int
-
-	// Args sets the arguments table used.
-	Args amqp.Table
-}
-
 // PublishSettings is the settings that will be used when a message is about
 // to be published to the message bus. These settings are only used by the
 // Client and never by the Server. For the server, Mandatory or Immediate
```

Comment:
- [ ] Update doc, Mandatory or Immediate is no longer supported by the ResponseWriter.